app/controllers/api/v1/books_controller.rb

```diff
@@ -2,7 +2,9 @@ module Api
   module V1
     class BooksController < ApplicationController
       def index
-        render_paginated Book.all
+        books = policy_scope(Book)
```

Comment:
This is OK! but as you are not filtering any book, and any user can see all the books, this can be removed

Reply:
Done